int/api/cmd/read_only_call_sc.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"encoding/base64"
+
+	"github.com/go-openapi/runtime/middleware"
+	"github.com/massalabs/station/api/swagger/server/models"
+	"github.com/massalabs/station/api/swagger/server/restapi/operations"
+	"github.com/massalabs/station/int/config"
+	"github.com/massalabs/station/pkg/node"
+	sendOperation "github.com/massalabs/station/pkg/node/sendoperation"
+	"github.com/massalabs/station/pkg/wallet"
+)
+
+func NewReadOnlyCallSCHandler(config *config.NetworkInfos) operations.CmdReadOnlyCallSCHandler {
+	return &ReadOnlyCallSC{networkInfos: config}
+}
+
+type ReadOnlyCallSC struct {
+	networkInfos *config.NetworkInfos
+}
+
+func (e *ReadOnlyCallSC) Handle(params operations.CmdReadOnlyCallSCParams) middleware.Responder {
+	args, err := base64.StdEncoding.DecodeString(params.Body.Args)
+	if err != nil {
+		return operations.NewCmdReadOnlyCallSCUnprocessableEntity().
+			WithPayload(
+				&models.Error{
+					Code:    errorInvalidArgs,
+					Message: err.Error(),
+				})
+	}
+
+	acc, err := wallet.Fetch(params.Body.Nickname)
+	if err != nil {
+		return operations.NewCmdReadOnlyCallSCBadRequest().WithPayload(
+			&models.Error{
+				Code:    errorInvalidNickname,
+				Message: "Error during wallet fetch: " + err.Error(),
+			})
+	}
+
+	if params.Body.Fee == "" {
+		params.Body.Fee = "0"
+	}
```

Comment:
What about setting the default value directly in the swagger file ? 
That would be simpler than setting default values in the code like that 🤔

Reply:
you're right

Reply:
but it doesn't work